treehouses/remote
# ssh & status: refresh (fixes #1248) (fixes #1302) (fixes #1309)

Fixes #1248 
Fixes #1302
Fixes #1309 

## How to Test #1248 and #1309 
1. Go to status.
2. Test the pull to refresh feature by pulling down on your screen. Everything should refresh and now it should also be able to display a more user friendly status update to user with "Checking..." instead of being static.
![Screenshot_20200807-194055_Treehouses Remote](https://user-images.githubusercontent.com/44847682/89700806-2e5ac600-d8e6-11ea-8069-e6c3791758f4.jpg)

## How to Test #1302 
1. Go to SSH.
2. Add a host if you dont have one under "Previously Connected Hosts" with the "Connect" button.
3. Click on a host's pencil icon to edit it.
4. Check to see that
    - changing the URI and saving it will instantly update the host list with the new name.
    - deleting the host with the trash button will instantly delete it from the host list.
![Screenshot_20200807-194148_Treehouses Remote](https://user-images.githubusercontent.com/44847682/89700802-2569f480-d8e6-11ea-92e3-a230ce94063e.jpg)
